mlflow/recipes/steps/train.py

```diff
@@ -453,24 +453,28 @@ def my_warn(*args, **kwargs):
             target_data = raw_validation_df[self.target_col]
             prediction_result = model.predict(raw_validation_df.drop(self.target_col, axis=1))
 
-            use_predict_proba = isinstance(prediction_result, pd.DataFrame)
-            if use_predict_proba and {
+            use_probability = False
```

Comment:
curious why we are renaming (i think?) `use_predict_proba` to `use_probability`. are we using the flag for something different now?

Reply:
Now its just used as an error function `use_probability` variable. So changed the naming.
I can make it better: `use_probability_for_error_function`. Is this better? What do you think?